wgsl/index.bs

```diff
@@ -6767,6 +6865,41 @@ The location of a function call is referred to as a <dfn noexport>call site</dfn
 Call sites are a [=dynamic context=].
 As such, the same textual location may represent multiple call sites.
 
+## Creation-time Functions ## {#creation-time-funcs}
+
+Function declarations annotated with the [=attribute/const=] attribute can be
+evaluated at [=shader module creation|shader-creation time=].
+These functions are called <dfn noexport>creation-time functions</dfn>.
+These functions can be part of [=creation-time expressions=].
+
+It is a [=shader-creation error=] if the function contains any expressions that
+are not [=creation-time expressions=], or any declarations that are not
+[=creation-time constants=].
+
+Note: The [=attribute/const=] attribute cannot be applied to user-declared functions.
+
+<div class='example wgsl' heading='Creation-time functions'>
+  <xmp>
+    @const fn maskBits(x : i32) -> i32 {
```

Comment:
This is not allowed because it's a user-defined function.

Should this instead use a builtin function and not in a comment that the builtin function is @const

Reply:
I rewrote this example using `firstLeadingBit` since it can be used as a creation-time expression, but does not support abstract integers.